routing/routing_benchmarks/bicycle_routing_tests.cpp

```diff
@@ -45,7 +45,7 @@ UNIT_CLASS_TEST(BicycleTest, Smoke)
   TestRouters(start, final);
 }
 
-UNIT_CLASS_TEST(BicycleTest, MailRuOfficeToUlSimanova5K2)
```

Comment:
It does give. Just have a look at osrm_turn_test.cpp. There're about 30 tests there. I wrote them more than one year ago but even now I can recall the test cases and theirs places by theirs names and theirs code for most of them. I can use lat lon coords for recovering the place as well but having such descriptive names I can take a short cut. Do you really think that address or short description is not useful at all? As for me it looks like a short comment for test which helps recall the test case quickly.